src/test/java/org/betonquest/betonquest/modules/logger/util/BetonQuestLoggerService.java

```diff
@@ -36,20 +40,22 @@
      * Default {@link BetonQuestLoggerService} Constructor.
      */
     public BetonQuestLoggerService() {
+        plugin = mock(Plugin.class);
```

Comment:
The Plugin mock should be reset() after every test.

```java
@BeforeEach
void prepareMocks() {
    when(plugin.getName()).thenReturn("GeneratedPlugin");
}

@AfterEach
void resetMocks() {
    reset(server);
}
```

An alternative would be to use the `@Mock` annotation:
https://javadoc.io/static/org.mockito/mockito-core/4.3.1/org/mockito/Mockito.html#mock_annotation